pkg/resource/certificate/sdk.go

```diff
@@ -152,6 +152,23 @@ func (rm *resourceManager) sdkCreate(
 	var resp *svcsdk.IssueCertificateOutput
 	_ = resp
 	resp, err = rm.sdkapi.IssueCertificateWithContext(ctx, input)
+	if err != nil {
+		input := &svcsdk.DescribeCertificateAuthorityInput{}
+		input.CertificateAuthorityArn = desired.ko.Spec.CertificateAuthorityARN
+
+		var describeResp *svcsdk.DescribeCertificateAuthorityOutput
+		describeResp, describeErr := rm.sdkapi.DescribeCertificateAuthorityWithContext(ctx, input)
+		rm.metrics.RecordAPICall("READ_ONE", "DescribeCertificateAuthority", err)
+		if describeErr != nil {
+			return nil, describeErr
```

Comment:
We should probably just requeue here as well, no reason to fail the resource on an internal error for describe.